src/App.js

```diff
@@ -120,6 +120,34 @@ function App() {
     SHOW_BACKGROUND: false,
```

Comment:
Overall, the code is well structured and easy to follow. However, there are a few improvements I would suggest:

1. **Use of magic numbers**: In the `decrementMintAmount` and `incrementMintAmount` functions, the numbers 1 and 10 are hardcoded. It would be better to replace these with named constants at the top of your code to improve readability and maintainability.

```jsx
const MIN_MINT_AMOUNT = 1;
const MAX_MINT_AMOUNT = 10;

...

const decrementMintAmount = () => {
  let newMintAmount = mintAmount - 1;
  if (newMintAmount < MIN_MINT_AMOUNT) {
    newMintAmount = MIN_MINT_AMOUNT;
  }
  setMintAmount(newMintAmount);
};

const incrementMintAmount = () => {
  let newMintAmount = mintAmount + 1;
  if (newMintAmount > MAX_MINT_AMOUNT) {
    newMintAmount = MAX_MINT_AMOUNT;
  }
  setMintAmount(newMintAmount);
};
```

2. **Error handling**: The `getConfig` function should have a `catch` block to handle any potential errors that may occur when fetching the config file.

```jsx
const getConfig = async () => {
  try {
    const configResponse = await fetch("/config/config.json", {
      headers: {
        "Content-Type": "application/json",
        Accept: "application/json",
      },
    });
    const config = await configResponse.json();
    SET_CONFIG(config);
  } catch (error) {
    console.error('Failed to load config:', error);
  }
};
```

3. **Repeated style properties**: There are repeated style properties in the styled components `StyledButton` and `StyledRoundButton`. It would be better to create a shared base styled component and extend it to avoid repetition.

```jsx
const ButtonBase = styled.button`
  padding: 10px;
  border: none;
  font-weight: bold;
  cursor: pointer;
  :active {
    box-shadow: none;
    -webkit-box-shadow: none;
    -moz-box-shadow: none;
  }
`;

const StyledButton = styled(ButtonBase)`
  border-radius: 50px;
  background-color: var(--secondary);
  color: var(--secondary-text);
  width: 100px;
  box-shadow: 0px 6px 0px -2px rgba(250, 250, 250, 0.3);
  -webkit-box-shadow: 0px 6px 0px -2px rgba(250, 250, 250, 0.3);
  -moz-box-shadow: 0px 6px 0px -2px rgba(250, 250, 250, 0.3);
`;

const StyledRoundButton = styled(ButtonBase)`
  border-radius: 100%;
  background-color: var(--primary);
  font-size: 15px;
  color: var(--primary-text);
  width: 30px;
  height: 30px;
  display: flex;
  align-items: center;
  justify-content: center;
  box-shadow: 0px 4px 0px -2px rgba(250, 250, 250, 0.3);
  -webkit-box-shadow: 0px 4px 0px -2px rgba(250, 250, 250, 0.3);
  -moz-box-shadow: 0px 4px 0px -2px rgba(250, 250, 250, 0.3);
`;
```

4. **Use of ternary operators**: The use of nested ternary operators in the return block can make the code hard to read. Consider breaking this out into smaller, well-named functions.